content/software-security/secure-software-development/minimum-attestation-references.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Minimum Attestation References"
+type: "article"
+date: 2023-05-10T15:21:01+02:00
+lastmod: 2023-05-10T15:21:01+02:00
+draft: false
+tags: ["Reference"]
+images: []
+menu:
+  docs:
+    parent: "secure-software-development"
+weight: 20
+toc: true
+---
+The minimum requirements within the Secure Software Attestation Form address requirements
+put forth in EO 14028 subsection (4)(e) and specific SSDF practices and tasks. For reference,
+please review the chart below.
+
+| Attestation Requirements | Related EO 14028 Subsection | Related SSDF Practices and Tasks |
+| -- | -- | -- |
+| 1) The software was developed and built in secure environments. Those environments were secured by the following actions, at a minimum: | 4e(i) | [See rows below]| 
+| a) Separating and protecting each environment involved in developing and building software; | 4e(i)(A) | [PO.5.1](/software-security/secure-software-development/ssdf/#PO.5.1) |
+| b) Regularly logging, monitoring, and auditing trust relationships used for authorization and access: i) to any software development and build environments; and ii) among components within each environment | 4e(i)(B) | [PO.5.1](/software-security/secure-software-development/ssdf/#PO.5.1) |
```

Comment:
```suggestion
| b) Regularly logging, monitoring, and auditing trust relationships used for authorization and access: i) to any software development and build environments; and ii) among components within each environment; | 4e(i)(B) | [PO.5.1](/software-security/secure-software-development/ssdf/#PO.5.1) |
```
Seems like a semicolon ought to go here, it's the only row in this column that doesn't end with one